### DISCUSSION TITLE
Allow uploading attachments to `render: shell` (or similar) render types

### DISCUSSION THREAD

**Discussion by altrisi** [community/community#4345] [Projects and Issues]
Just like the example suggests, the `render: shell` is a good way to ask for log information in an issue form. While being able to paste such log output is nice, a problem arises when a larger log file is needed: There is no way to attach a file in that textarea.

I think there should be a way to allow file attachments in those type of form elements while still formatting them properly in case the log output was added directly in there.

Thanks!

_PS: Do note that there is an upvote button other than the thumbs up button, that is more likely to drive attention if you think the feature would be useful_

**Comment by AlCalzone** [comment]
I second this. We require users to **upload** logfiles. If they are short it is okay to post them as text, but currently there's no way to automatically format them and have the upload function at the same time.

**Comment by lukehefson** [comment]
Related: https://github.com/github/feedback/discussions/4291
